src/ui/public/filter_bar/lib/map_geo_bounding_box.js

```diff
@@ -1,20 +1,33 @@
 import _ from 'lodash';
+import { SavedObjectNotFound } from '../../errors';
 
 export function FilterBarLibMapGeoBoundingBoxProvider(Promise, courier) {
   return function (filter) {
     if (filter.geo_bounding_box) {
-      return courier
-      .indexPatterns
-      .get(filter.meta.index).then(function (indexPattern) {
+      function getParams(indexPattern) {
         const type = 'geo_bounding_box';
         const key = _.keys(filter.geo_bounding_box)
           .filter(key => key !== 'ignore_unmapped')[0];
-        const field = indexPattern.fields.byName[key];
         const params = filter.geo_bounding_box[key];
-        const topLeft = field.format.convert(params.top_left);
-        const bottomRight = field.format.convert(params.bottom_right);
+        const topLeft = indexPattern
+        ? indexPattern.fields.byName[key].format.convert(params.top_left)
+        : JSON.stringify(params.top_left);
+        const bottomRight = indexPattern
+         ? indexPattern.fields.byName[key].format.convert(params.bottom_right)
+         : JSON.stringify(params.bottom_right);
```

Comment:
Should these be indented by two?
```
  const bottomRight = indexPattern
    ? indexPattern.fields.byName[key].format.convert(params.bottom_right)
    : JSON.stringify(params.bottom_right);
```

Same below, I think the `.thens` are supposed to be indented by two because it's a continuation of the original line.

Reply:
Fixed the ternary expressions, but the `thens` are a separate matter. This style guide rule was changed at some point but the code base was never reformatted en masse. I'm following the existing style in these files in order to avoid having inconsistent styles within a single module.